.github/workflows/main.yml

```diff
@@ -29,11 +29,13 @@ jobs:
           fetch-depth: 50
 
       - name: Install Rust Stable
+        env:
+          RUST_VERSION: "1.85.0"
```

Comment:
If we are pinning the Rust version, can we do so with a `rust-toolchain.toml` rather than an environment variable? The way the variable is setup right now is quite hard to discover...

Reply:
Can I do it in another PR as it's unrelated to this one? This is just a small refactoring to avoid specifying the version twice.

Reply:
The version was already pinned in CI

Reply:
Oh, I missed that we were already pinning it. Then sure.